src/components/Layouts/Primary/NavBar/NavTools/ProfileMenuPanel/ProfileMenuPanel.tsx

```diff
@@ -99,6 +71,8 @@ export const ProfileMenuPanel: React.FC = () => {
     });
   };
 
+  const { panelPages: addProfileContent }: { panelPages } = useNavPages(false);
```

Comment:
This is actually making the type of `panelPages` `any`.

```suggestion
  const { panelPages: addProfileContent } = useNavPages(false);
```

Reply:
I actually realized that it does have an `onClick` in props. I added that interface back.

`{addProfileContent.map(({ title, href, onClick }, index) => (`